src/components/wizard/Wizard.jsx

```diff
@@ -93,7 +93,7 @@ const Wizard = () => {
     if (
       stepIndex > currentStep &&
       !stepData[stepIndex].visited &&
-      !options.enableForwardSkip
+      !options.enableWizardStepSkip
```

Comment:
Just make sure that `options.enableForwardSkip` is not used for incremental show of questions, because this way renaming would not make sense.

Reply:
Yes, it is not used anywhere else than for the wizard steps.